docs/_embed/zap_editor.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Zap Editor
 order: 4
-layout: post
+layout: post-toc
```

Comment:
This will auto-generate the righthand table of contents nav that the page is currently missing. Will make it easier to share anchor links to Partners and link to a specific section on other docs.